vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -239,7 +240,7 @@ You can alter the icons used by specifying the name of the icon in the [Google M
                     items=[
                         vm.NavLink(
                             label="Section 1",
-                            icon="bar_chart",
+                            icon="Bar Chart",
```

Comment:
Can we revert to "bar_chart" as "Bar chart" is not a valid google icons name, and leads to error in loading the icon from the library.

Reply:
Hey Nadija, the newly added validator converts "Bar Chart" to "bar_chart", so both of them produce the same results.

Reply:
Got it. I tested on my local branch. Sorry.